### ISSUE TITLE
Power mode makes quickblades good weapon

### ISSUE THREAD

**Issue by Sandman25DCSS** [jeremygurr/dcssca#454]
I tried Vaults 5 without Power mode and I see that quickblade is no longer a good weapon, it barely does any damage even with +5 slaying. While in Power mode it is really great.
I suspect it happens because Power mode increases damage before AC. Not sure if we want to change balance of quick blades

**Comment by jeremygurr** [comment]
I'm not sure what's going on here. If you can tell me the name of a character that is in this situation so I can test it I can better understand. The power mode enhancement is applied before monster ac of course, like all other enhancers (including slaying). Do we not want this?

Seems like an occasional situation that forces you to use power mode for a weapon to be effective isn't necessarily a bad thing.

**Comment by yrmvgh** [comment]
that's an interesting point, about the difference between damage(power)-ac = final, and damage-ac=final(power). That does impact low damage weapons a lot more than their heavy counterparts, effectively increasing their actual damage output by a larger number

**Comment by Sandman25DCSS** [comment]
I quit the character with 3 runes after realizing that it is not viable in normal mode, it felt like some cheating.
It was a Sp (http://crawl.homedns.org/morgue/Sandman25/morgue-Sandman25-20160509-013834.txt) with +9 quickblade of protection and 0-10 slaying. In normal mode it took many attacks to kill monsters with good AC, in power mode it was much better, especially when berserking.

**Comment by yrmvgh** [comment]
Maybe it's desirable to take a character that's normally not viable and turn it playable?

**Comment by Sandman25DCSS** [comment]
If it's the case then the character is one of the best examples. In normal mode I couldn't cast Invisibility, Deflect Missiles, stab monsters, melee, escape. I mean I almost never was in normal mode.

**Comment by jeremygurr** [comment]
Thinking about this a lot, it doesn't make sense that power mode would increase post-ac damage. Power mode represents the idea of attacking with greater ferocity, expending more energy in the process. So if that means that a quickblade that does 30% more damage, but manages to cause 2x post ac damage, because it's just past the ac damage shaving threshold, then that makes sense to me. 

If power mode feels too powerful, then maybe the cost should be a little higher. Right now the cost is 2 SP per action, but we could easily make that 3 or 4, so it still may only be used for a limited time. And the consequence of running out of SP is much higher than before. Previously, if you were out of SP, it meant your escape options were reduced significantly. But for a melee fighter that often wasn't a big deal. Certainly doing more damage was worth the risk. But now, when you run out of SP, your fighter attacks at 75% damage and attack speed, creating a very undesirable situation, making normal mode much more appealing for longer battles, and power mode effective only for shorter engagements. I think I will take the SP costs to 3 for power mode. This is especially significant for magic users who wish to spam power mode attack spells. 

In short, I think your +9 quickblade _should_ be devastatingly powerful in the situation you described, but for a shorter period of time, and at a higher risk to the player. You certainly shouldn't be able to just stay in power mode for long periods of time and while facing dangerous enemies. At least it shouldn't be a no brainer decision. The risks should be carefully evaluated for different situations.

**Comment by Sandman25DCSS** [comment]
If stamina cost is 2 for every attack no matter the weapon type, then it's a big nerf of all low base damage weapons. 2h weapons were already basically recommended by almost everyone, now 1h + shield is even weaker.

**Comment by jeremygurr** [comment]
For a melee character, there are basically two stamina costs to worry about.

Normal attack cost: which is based on strength, fighting, and weapon weight (two handed weapons are heavier than one handed). This cost is taken no matter what exertion mode the player has selected.

Power or quick mode costs: A player in power mode has an additional 3 SP per attack cost, no matter the weapon or attributes of the player. And if they have Quick mode active as well, that's a added 3 SP per movement cost. 

The weapon weight increasing SP costs allows lighter, single handed weapons to be able to attack for longer at full strength, or in power mode longer, than a player with a two handed weapon could, especially if a player wasn't very strong. 

On the other hand, since power mode is based on number of attacks, players who have slower attacks will have less of a consequence from being in power mode, since the price isn't extracted as often. But since their normal mode SP per attack price is higher anyway, maybe it somewhat balances out. 

If needed, we can increase the effect of weapon weight to shift the balance of power a little away from two handed wielders and make 1 handed a more appealing option than it is now. Ideally, they ought to be close to balanced as far as what the best long term equipment configuration is.
